examples/inspection/plot_permutation_importance.py

```diff
@@ -130,8 +131,8 @@
 y_ticks = np.arange(0, len(feature_names))
 fig, ax = plt.subplots()
 ax.barh(y_ticks, tree_feature_importances[sorted_idx])
+ax.yaxis.set_major_locator(FixedLocator(y_ticks))
 ax.set_yticklabels(feature_names[sorted_idx])
```

Comment:
```suggestion
ax.set_yticks(y_ticks)
ax.set_yticklabels(feature_names[sorted_idx])
```

Reply:
I prefer the following. It avoids importing `FixedLocator` that would be potentionally an unknown class from most of our users.